e2e/_suites/fleet/services.go

```diff
@@ -498,7 +498,7 @@ func newTarInstaller(image string, tag string) (ElasticAgentInstaller, error) {
 		InstallFn:         installFn,
 		installerType:     "tar",
 		logFile:           "elastic-agent.log",
-		logsDir:           "/opt/Elastic/Agent/data/elastic-agent-%s/logs/",
```

Comment:
I would prefer looking into `/opt/Elastic/Agent/data/elastic-agent-{hash}/logs/` for `elastic-agent-json.log`

Reply:
We are `cat`-ing this file when errors are thrown. Must that be changed?

See https://github.com/elastic/e2e-testing/blob/7c2b5c2e91bc6f290c969c07a39fd572bd6f2a99/e2e/_suites/fleet/services.go#L115

Reply:
Well, both files do exist, and can be `cat-ted`, but my point is: what format will be more readable in the logs (both CI and local terminal)?

### JSON format
```shell
{"log.level":"debug","@timestamp":"2020-12-01T09:48:14.393Z","log.origin":{"file.name":"kibana/client.go","file.line":170},"message":"Request method: POST, path: /api/fleet/agents/ffa27e57-15e3-4696-8e48-09900f6a1856/acks","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2020-12-01T09:48:15.044Z","log.origin":{"file.name":"log/reporter.go","file.line":40},"message":"2020-12-01T09:48:15Z: type: 'STATE': sub_type: 'RUNNING' message: Application: metricbeat--7.10.1[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2020-12-01T09:48:15.185Z","log.origin":{"file.name":"log/reporter.go","file.line":40},"message":"2020-12-01T09:48:15Z: type: 'STATE': sub_type: 'RUNNING' message: Application: filebeat--7.10.1--36643631373035623733363936343635[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2020-12-01T09:48:15.631Z","log.origin":{"file.name":"log/reporter.go","file.line":40},"message":"2020-12-01T09:48:15Z: type: 'STATE': sub_type: 'RUNNING' message: Application: metricbeat--7.10.1--36643631373035623733363936343635[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2020-12-01T09:48:16.016Z","log.origin":{"file.name":"log/reporter.go","file.line":40},"message":"2020-12-01T09:48:16Z: type: 'STATE': sub_type: 'RUNNING' message: Application: filebeat--7.10.1[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running","ecs.version":"1.6.0"}
{"log.level":"debug","@timestamp":"2020-12-01T09:48:16.271Z","log.origin":{"file.name":"application/fleet_gateway.go","file.line":162},"message":"FleetGateway is sleeping, next update in 1s","ecs.version":"1.6.0"}
{"log.level":"debug","@timestamp":"2020-12-01T09:48:17.673Z","log.origin":{"file.name":"application/fleet_gateway.go","file.line":142},"message":"FleetGateway calling Checkin API","ecs.version":"1.6.0"}
{"log.level":"debug","@timestamp":"2020-12-01T09:48:17.674Z","log.origin":{"file.name":"kibana/client.go","file.line":170},"message":"Request method: POST, path: /api/fleet/agents/ffa27e57-15e3-4696-8e48-09900f6a1856/checkin","ecs.version":"1.6.0"}
```

### Log format
```shell
2020-12-01T09:48:14.393Z        DEBUG   kibana/client.go:170    Request method: POST, path: /api/fleet/agents/ffa27e57-15e3-4696-8e48-09900f6a1856/acks
2020-12-01T09:48:15.044Z        INFO    log/reporter.go:40      2020-12-01T09:48:15Z: type: 'STATE': sub_type: 'RUNNING' message: Application: metricbeat--7.10.1[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running
2020-12-01T09:48:15.185Z        INFO    log/reporter.go:40      2020-12-01T09:48:15Z: type: 'STATE': sub_type: 'RUNNING' message: Application: filebeat--7.10.1--36643631373035623733363936343635[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running
2020-12-01T09:48:15.631Z        INFO    log/reporter.go:40      2020-12-01T09:48:15Z: type: 'STATE': sub_type: 'RUNNING' message: Application: metricbeat--7.10.1--36643631373035623733363936343635[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running
2020-12-01T09:48:16.016Z        INFO    log/reporter.go:40      2020-12-01T09:48:16Z: type: 'STATE': sub_type: 'RUNNING' message: Application: filebeat--7.10.1[ffa27e57-15e3-4696-8e48-09900f6a1856]: State changed to RUNNING: Running
2020-12-01T09:48:16.271Z        DEBUG   application/fleet_gateway.go:162        FleetGateway is sleeping, next update in 1s
2020-12-01T09:48:17.673Z        DEBUG   application/fleet_gateway.go:142        FleetGateway calling Checkin API
2020-12-01T09:48:17.674Z        DEBUG   kibana/client.go:170    Request method: POST, path: /api/fleet/agents/ffa27e57-15e3-4696-8e48-09900f6a1856/checkin
```

Reply:
@michalpristas any preferred option here?

Reply:
Hey @michalpristas, I'm gonna merge this one. Please open a new issue if we want to use the JSON logs. Thanks!